.gitignore

```
@@ -5,5 +5,8 @@ tmp
 # Goreleaser build folder
 dist/
 
+# build file with `make build`
+yamlfmt
```

Might as well add this for Windows while we're at it

```suggestion
yamlfmt
yamlfmt.exe
```

---

Good point👍 
Applied the commit as https://github.com/google/yamlfmt/pull/120/commits/16e74b7bf0ca8786e6ecf9ca57b99e542dd5da58